### DISCUSSION TITLE
Welcome to JayantGoel001 Discussions!

### DISCUSSION THREAD

**Discussion by JayantGoel001** [JayantGoel001/JayantGoel001#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JayantGoel001** [comment]
![bgimg](https://user-images.githubusercontent.com/54479676/105956598-0df21400-609e-11eb-95f5-4d1d2ab6fab9.jpg)

**Reply by bulldog5426** [reply]
Hey, I wanted to ask how come you are so active on github.Seems like you spend 10 hors a day over git only. Like lietrally man live for once,get a life dude.Fucking Crazyyyyyyyyyy!!!!!

**Reply by JayantGoel001** [reply]
I am not as much active as you are thinking and seeing 😅

**Comment by bulldog5426** [comment]
Lmao dead

**Comment by bulldog5426** [comment]
Spammer alert

**Comment by bulldog5426** [comment]
Katherine Pierce is back guyss!!!!!!!! I need blood blood blood .. You guys are safe just because of daylight.Coming after you at night!!!!!!
